--- b/spinal/coregen/MIG/user_design/rtl/MIG.v
+++ a/spinal/coregen/MIG/user_design/rtl/MIG.v
@@ -285,7 +285,7 @@
    localparam C1_CLKOUT1_DIVIDE       = 1;       
    localparam C1_CLKOUT2_DIVIDE       = 16;       
    localparam C1_CLKOUT3_DIVIDE       = 8;       
+   localparam C1_CLKFBOUT_MULT        = 5;       
-   localparam C1_CLKFBOUT_MULT        = 2;       
    localparam C1_DIVCLK_DIVIDE        = 1;       
    localparam C1_ARB_ALGORITHM        = 0;       
    localparam C1_ARB_NUM_TIME_SLOTS   = 12;       
@@ -407,7 +407,7 @@
    localparam C3_CLKOUT1_DIVIDE       = 1;       
    localparam C3_CLKOUT2_DIVIDE       = 16;       
    localparam C3_CLKOUT3_DIVIDE       = 8;       
+   localparam C3_CLKFBOUT_MULT        = 5;       
-   localparam C3_CLKFBOUT_MULT        = 2;       
    localparam C3_DIVCLK_DIVIDE        = 1;       
    localparam C3_ARB_ALGORITHM        = 0;       
    localparam C3_ARB_NUM_TIME_SLOTS   = 12;       
